posit_team_applications_faq.md

```diff
@@ -32,6 +32,10 @@ You can then install the `{phsmethods}` package and all of its other dependencie
 install.packages("phsmethods")
 ```
 
+#### I can't install / use a package because it needs `{rJava}`
+
+Some packages, e.g. `{xlsx}`, `{XLconnect}`, depend on an installation of [Java](https://en.wikipedia.org/wiki/Java_(software_platform)) in order to work.  There are no plans to support Java on Posit Workbench.  Alternative packages such as `{openxlsx}` that do not rely on Java can be used instead.
```

Comment:
....... no current or future plans to support Java on Posit Workbench.  Alternative packages such as `{openxlsx}` that do not rely on Java should be used instead.

Reply:
@CliveWG I've updated as per your suggested wording.  Thanks!